code/game/objects/items/tools/crowbar.dm

```diff
@@ -56,6 +56,42 @@
 	item_state = "crowbar_large"
 	toolspeed = 0.5
 
+/obj/item/crowbar/engineering
+	name = "engineering crowbar"
+	desc = "It's a big crowbar, perfect for fending off those assistants trying to get at your gloves."
+	force = 12
+	//w_class = WEIGHT_CLASS_NORMAL Commented out so it can fit in belts
+	throw_speed = 3
+	throw_range = 3
+	materials = list(MAT_METAL = 400)
+	icon_state = "crowbar_eng"
+	item_state = "crowbar_eng"
+	belt_icon = "crowbar_eng"
+	toolspeed = 0.5
```

Comment:
```suggestion
	toolspeed = 0.5
```
This seems a bit strong for a roundstart tool, although I'm no balance nerd

Reply:
This is the speed the large crowbar has by default, but I'm fine with changing it if needed

Reply:
oh, my bad. Why the hell is the default value for this tool 0.5, what a meme.